avocado_vt/plugins/vt_list.py

```diff
@@ -96,22 +98,44 @@ def configure(self, parser):
 
         vt_compat_group_lister = list_subcommand_parser.add_argument_group(
             'Virt-Test compat layer - Lister options')
-        vt_compat_group_lister.add_argument("--vt-list-guests",
-                                            action="store_true",
-                                            default=False,
-                                            help="Also list the available "
-                                            "guests (this option ignores the "
-                                            "--vt-config and --vt-guest-os)")
-        vt_compat_group_lister.add_argument("--vt-list-archs", default=False,
-                                            action="store_true",
-                                            help="Also list the available "
-                                            "arch/machines for the given "
-                                            "guest OS. (Use \"--vt-guest-os "
-                                            "''\" to see all combinations; "
-                                            "--vt-config --vt-machine-type "
-                                            "and --vt-arch args are ignored)")
+
+        help_list_guests = ("Also list the available guests (this option "
+                            "ignores the --vt-config and --vt-guest-os)")
+        help_list_archs = ("Also list the available arch/machines for the "
+                           "given guest OS. (Use \"--vt-guest-os ''\" to see "
+                           "all combinations; --vt-config --vt-machine-type "
+                           "and --vt-arch args are ignored)")
+        if hasattr(Settings, 'register_option'):
```

Comment:
This seems like another good candidate for a `virttest.compat` utility function.